code/lesson2/1_basic_animation.R

```diff
@@ -7,18 +7,20 @@ library(ggmap)
 library(tidyverse)
 
 
-setwd("YOUR/PATH/TO/data/act")
+setwd("/YOUR/PATH/TO/data/otn")
+
+unzip('nsbs_matched_detections_2022.zip', overwrite = TRUE)
```

Comment:
just a note, that dplyr's read_csv() can also pull in the zip as a CSV (it will correctly identify which file to pull in) and then the variable created by read_csv() could be passed to read_otn_detections() I think! don't care which way you do it haha just a note